requirements.txt

```diff
@@ -0,0 +1,17 @@
+Jinja2==2.8
+MarkupSafe==0.23
+PyYAML==3.11
+ansible==1.9.4
+cffi==1.5.0
+cryptography==1.2.1
+ecdsa==0.13
+enum34==1.1.2
+idna==2.0
+ipaddress==1.0.16
+paramiko==1.16.0
+pyOpenSSL==0.15.1
+pyasn1==0.1.9
+pycparser==2.14
+pycrypto==2.6.1
+six==1.10.0
+wsgiref==0.1.2
```

Comment:
Should we leave versions out of the requirements except for the ansible version? Most of these are dependencies of ansible. Can we limit the requirements to ansible and pyopenssl?